scripts/gen_profiles_layer.py

```diff
@@ -351,14 +351,14 @@
 ENUMERATE_ALL = '''
 // Get all elements from a vkEnumerate*() lambda into a std::vector.
 template <typename T>
-VkResult EnumerateAll(std::vector<T> *vect, std::function<VkResult(uint32_t *, T *)> func) {
+VkResult EnumerateAll(std::vector<T> &vect, std::function<VkResult(uint32_t *, T *)> func) {
```

Comment:
This is cosmetic, I thing if we are doing performance optimizations it should not be mixed up with cosmetic changes,

Reply:
Its a general style to use references when passing into functions, so while cosmetic in a sense, it is more in line with the general style.